### ISSUE TITLE
Weekly testing of user guide notebooks

### ISSUE THREAD

**Issue by hakonanes** [pyxem/orix#330]
The new weekly testing of user guide notebooks using [nbval](https://nbval.readthedocs.io/en/latest/) works as expected. The action log is [here](https://github.com/pyxem/orix/runs/6346988629?check_suite_focus=true).

The only notebook with cell outputs stored is the "Clustering orientations" one. I would expect all notebooks that haven't stored cell outputs to pass, however some cells in the two other clustering notebooks fail. I'll look into this when I find the time.

**Comment by harripj** [comment]
Look like `sklearn` needs to be installed when using the action: https://github.com/pyxem/orix/runs/6346988629?check_suite_focus=true#step:6:1926

I will add this to the action and run again.

**Comment by hakonanes** [comment]
Good point. I'll fix this now, as I have to fix `setup.py` for uploading to PyPI.

**Comment by hakonanes** [comment]
Fantastic, everything works ([log](https://github.com/pyxem/orix/runs/6371223381?check_suite_focus=true)) except the cell in the "Clustering orientations" notebook printing the time for `get_distance_matrix()`, which we shouldn't expect to be the same for my machine (slightly faster) and the GitHub runner.
